utils/animations.ts

```diff
@@ -0,0 +1,78 @@
+import anime from 'animejs';
+import {Polygon, PolygonPoints} from '../components/Shape';
+import styles from '../styles/lobby.module.scss';
+
+const DEFAULTS = {
+  easing: 'easeInOutCubic',
+  delay: 0,
+};
+
+function morphPolygon(
+  target: HTMLElement,
+  shape: Polygon,
+  forwards: boolean,
+): void {
```

Comment:
can we have these be options? the parameters have a lot in common and I think the following model will be helpful for the other devs

```ts
interface BaseAnimationOptions {
  readonly target: HTMLElement;
  readonly forwards: boolean;
}

interface MorphPloygonOptions extends BaseAnimationOptions {
  readonly shape: Polygon;
}

interface AnimatePloygonOptions extends MorphPolygonOptions {
  readonly polygonNode: HTMLElement;
}
```